src/Symfony/Component/Cache/Adapter/AbstractAdapter.php

```diff
@@ -336,19 +335,9 @@ public function __destruct()
         }
     }
 
-    private function getId($key, $ns = false)
+    private function getId($key)
```

Comment:
This would just raise the boilerplate, whereas the current behavior keep it low IMHO.